.github/workflows/tests.yml

```diff
@@ -70,6 +70,13 @@ jobs:
         pip install --use-deprecated=legacy-resolver scikit-optimize
         pip install --use-deprecated=legacy-resolver cma
 
+    # TODO: Add the deselected tests back after fixing version constraint of sklearn. 
```

Comment:
A nit, but please remove the trailing whitespace.
```suggestion
    # TODO: Add the deselected tests back after fixing version constraint of sklearn.
```

Reply:
Sorry about that 🙇

Reply:
No problem. Thank you for your swift action!